ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -167,7 +168,10 @@ private Response handleQuery(
     return Response.ok().entity(queryStreamWriter).build();
   }
 
-  private Response handlePrintTopic(final PreparedStatement<PrintTopic> statement) {
+  private Response handlePrintTopic(
```

Comment:
nit: can you follow the pattern of `handlQuery` and just pass the `streamingProperties` down as the second param?

e.g. 

```
private Response handlePrintTopic(
   final PreparedStatement< PrintTopic > statement,
      final Map<String, Object> streamsProperties
  ) {
```